specs/UI_import_gluster_cluster_workflow.adoc

```diff
@@ -0,0 +1,218 @@
+:imagesdir: ./images/
+
+= Import gluster cluster workflow
+
+== Problem description
+
+This specification describes import gluster cluster workflow which is a mechanism to add an gluster storage cluster into tendrl for management.
+
+View design will look like:
+
+image::import.png[Import workflow]
+
+== Use Cases
+
+This addresses the use case of importing existing gluster clusters into tendrl for management.
+
+The workflow is as follows:
+
+* API will provide the list of clusters(imported and ready to import) and those clusters will be displayed on cluster list page.
+
+* A button will displayed for each cluster which are not imported and are ready to import.
+
+* User will click on the import button for the cluster, he wants to import.
+
+* User will be redirected to new view.
+
+* UI will filter the available nodes for importing the cluster.
+
+* The view will list down the available nodes and a checkbox for enabling/disabling the cluster wide volume profiling.
+
+* Each node's name, IP address and role will be shown.
```

Comment:
What IP is showed if node has multiple ip?

There is not such thing as node name, you can use "clusters/<cluster_id>/nodes/<node_id>/NodeContext/fqdn"

Roles can be picked up from ""clusters/<cluster_id>/nodes/<node_id>/NodeContext/tags"  (it is a json array)